app/src/main/java/com/infomaniak/mail/ui/main/folder/TwoPaneFragment.kt

```diff
@@ -133,15 +133,20 @@ abstract class TwoPaneFragment : Fragment() {
         }
     }
 
-    fun navigateToThread(thread: Thread) = with(twoPaneViewModel) {
+    fun navigateToThread(thread: Thread) {
         if (thread.isOnlyOneDraft) {
             trackNewMessageEvent(OPEN_FROM_DRAFT_NAME)
-            openDraft(thread)
+            twoPaneViewModel.openDraft(thread)
```

Comment:
The ViewModel shouldn't do UI related stuff.

Reply:
There no UI stuff here 🤔